acstools/acs_destripe.py

```diff
@@ -22,19 +22,13 @@
 Examples
 --------
 
-In Python without TEAL (recommended):
+In Python (recommended):
```

Comment:
Again, no need to include "(recommended)" as using it via Python is the only option

Reply:
Same as above. It has a CLI.